plugins/rich-editor/src/scripts/blots/EmojiBlot.js

```diff
@@ -17,5 +17,6 @@ export default class EmojiBlot extends Embed {
     }
 }
 
-EmojiBlot.blotName = 'emoji';
+EmojiBlot.className = "emoji";
+EmojiBlot.blotName = 'emojis';
```

Comment:
Why isn't it the same as the class name? I'd rather keep plural for groups or lists, this is a singular blot.